lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -224,4 +232,33 @@ describe('BasicLtiLaunchApp', () => {
 
     assert.notCalled(fakeApiCall);
   });
+
+  context('when lmsGrader is mode is true', () => {
+    beforeEach(() => {
+      fakeConfig.lmsGrader = true;
+      fakeConfig.grading = {
+        students: [{ username: 'user1' }, { username: 'user2' }],
+      };
+    });
+
+    it('renders the LMSGrader component', () => {
+      const wrapper = renderLtiLaunchApp();
+      const LMSGrader = wrapper.find('.LMSGrader');
```

Comment:
I would suggest to mock the `LMSGrader` component here as we do for `Dialog`, `ErrorDisplay` etc. above. Otherwise the side-effects/implementation details of that component end up complicating these tests. For example, here you've had to mock `document.querySelector` even though the `BasicLtiLaunchApp` component doesn't touch that as its a concern of the `LMSGrader` component.